core-models/src/main/scala/com/mesosphere/usi/core/models/PodSpec.scala

```diff
@@ -1,7 +1,15 @@
 package com.mesosphere.usi.core.models
 
 /**
-  * Specification of a pod.
-  * @param id Id of the specification.
+  * Specification of some Pod that should be launched.
+  *
+  * Pods are launched at-most-once. It is illegal to transition of podSpec from goal terminal to goal running.
```

Comment:
```suggestion
  * Pods are launched at-most-once. It is illegal to transition of pod spec from goal terminal to goal running.
```

We should agree on one way to spell it. Camel case is fine with me.